src/content/developers/tutorials/send_token_etherjs.md

```diff
@@ -0,0 +1,71 @@
+## send Token Using ethers.js(5.0)
+
+this send_token() is not only for ether transfer 
+but also for other erc20 tokens too.
+
+### Prerequisite
+1. include ethers.js(5.0) - it might not be necessary to implement it with version code but the code below is version 5.0
+
+### Parameter
+1. contract_address - token contract address(contract address is needed when the token you want to transfer is not ether(any other erc20 token)))
+2. send_token_amount - the amount you want to send to the receiver
+3. to_address - the receiever's address
```

Comment:
"receiver's"

Reply:
Thanks for your precise feedback.
I revised the tutorial(commit).
Fixed grammar errors and added metadata, etc.
Please let me know if there are other parts that needs to revise to be used in your site.